assets/src/edit-story/elements/utils/visibleImage.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import useBrightnessChecker from './useBrightnessChecker';
+
+const Image = styled.img`
+	${ ( { isTooBright } ) => isTooBright && `
+		/* Using filter rather than box-shadow to correctly follow
+		 * outlines in semi-transparent images like gif and png.
+		 */
+		filter: drop-shadow( 0 0 5px rgba(0, 0, 0, 0.5) );
```

Comment:
Does dropping shadow in all cases look really bad?

Reply:
I just followed the Figma. I haven't tried having it on always for all images, but suspect it might be a little confusing.